### DISCUSSION TITLE
Stable replacement for grafonnet-lib

### DISCUSSION THREAD

**Discussion by jacksongoode** [grafana/grafonnet#94] [Q&A]
> This library is experimental

Just wondering if there's are outstanding bugs/instabilities that prevent recommending migration from granfonnet-lib? My team and I would like to eventually port over from @rhowe 's fork.

**Comment by Duologic** [comment]
There are no outstanding bugs/instabilities that prevent recommending it. Note that we have marked it as experimental as this is in development and might change without much notice, it also doesn't receive any support through the official channels. We give repo-level support through the issues and discussions here on the repository and if you find a bug or get stuck we'll attempt to fix it as soon as possible but without giving guarantees.

Or in other words, we deliver best-effort support at this point, I suppose at a similar level as @rhowe's fork.

**Reply by rhowe** [reply]
I'd say this is better supported than my fork - I am merging dependabot MRs but other than that it's not progressing much. I would encourage people to try moving to the new library instead, although I admit I haven't done that myself yet.

**Reply by jacksongoode** [reply]
Thanks for the response @rhowe and to get your vote of confidence. I think the big migration trouble will be the changing syntax but I think there might be a tool that Duologic prototyping that might allow some scaffolding from JSON => Graffonnet initially.

**Reply by Duologic** [reply]
Note I'm not actively working on the JSON to jsonnet tooling, also see #32 for further discussion on that.
